utility/src/main/java/org/oppia/android/util/logging/EventBundleCreator.kt

```diff
@@ -886,6 +886,7 @@ class EventBundleCreator @Inject constructor(
       ScreenName.ONBOARDING_PROFILE_TYPE_ACTIVITY -> "onboarding_profile_type_activity"
       ScreenName.CREATE_PROFILE_ACTIVITY -> "create_profile_activity"
       ScreenName.INTRO_ACTIVITY -> "intro_activity"
+      ScreenName.FEATURE_FLAG_ACTIVITY -> "feature_flag_activity"
```

Comment:
Ditto: to consider plural forms:

```suggestion
      ScreenName.FEATURE_FLAGS_ACTIVITY -> "feature_flags_activity"
```

Reply:
Updated